spec/operators/filter-spec.ts

```diff
@@ -272,4 +272,36 @@ describe('Observable.prototype.filter', () => {
     expectObservable(r, unsub).toBe(expected);
     expectSubscriptions(source.subscriptions).toBe(subs);
   });
-});
+
+  it('should not be compile error', () => {
+    // tslint:disable no-unused-variable
```

Comment:
It's possible that a later version of TypeScript compiler might completely remove code related to unused variables as an optimization. I'm completely speculating, but it might be worth using `expect()` to set some obvious assertions around the variables you're not using, and remove these `tslint:disable` comments.